CHANGELOG.rst

```diff
@@ -32,11 +32,12 @@ Deprecations:
 Changes:
 ^^^^^^^^
 
+
+- Add support for static typing when using ``time``, ``count_exceptions`` and ``track_inprogress`` as decorators.
 - Added support for `OpenMetrics <https://openmetrics.io>`_ exposition in ``prometheus_async.aio.web.server_stats()`` and thus ``prometheus_async.aio.web.start_http_server_in_thread()``.
   `#23 <https://github.com/hynek/prometheus-async/issues/23>`_
 
 
-
```

Comment:
pls remove this

Reply:
What am I removing here? The empty line?

Reply:
no…you're removing the absence of an empty line :D